web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,47 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+### OPTION-1: Kebab menu option:
+
+1. Login to Openshift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes” [list view]
+3. There will be a label which indicates which SC is a default one.
+4. User selects the SC that he/she would like to make it as a ‘default class’
+5. Click on the kebab menu > select “Edit Default Storage Class” as soon in the screen below:
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_opt_1.png)
+
+6. Pop-up opens with a list of ‘SC’ and message shown below:
+*Message:
+Current default storage class will be changed to false when you
+choose a different storage class from the list below.**
+
+7. User can choose any SC as a default storage class and click on ‘Save’ button to confirm the change.
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_popUp_1.png)
+
+### OPTION-2: Change the ‘default’ option from ‘detail page’:
+
+1. Login to Openshift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. Click on SC name link (shown as link)
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_opt_2.png)
+
+3. SC details are shown as below
+4. Click on “pencil icon” near the ‘default class status’
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_opt_2_detailsPg.png)
+
+5. Pop-up opens with a list of ‘SC’ and message shown below:
+** Message:
+Current default storage class will be changed to false when you
+choose a different storage class from the list below.**
+
+6. User can choose any SC as a default storage class and click on ‘Save’ button to confirm the change.
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_popUp_2.png)
```

Comment:
would it be helpful to have it read "gp2 (current default)" to align with text above?
Also, nitpic but these designs should align with PF4 dropdowns/buttons.

Reply:
@matthewcarleton @muruGanesan 

We've a previous design to say "gp2 (default)" as part of improvements to Storage Class listing.

@muruGanesan - can you please check with @yuvalgalanti on this?  I know we faced resistance in a PR (implementation) to get this approved previously.